apps/src/calc/calc.js

```diff
@@ -80,6 +81,12 @@ function getTokenList(one, two) {
   if (two instanceof Equation) {
     two = two.expression;
   }
+  if (jsnums.isSchemeNumber(one)) {
+    one = new ExpressionNode(one);
+  }
+  if (jsnums.isSchemeNumber(two)) {
+    two = new ExpressionNode(two);
+  }
   if (typeof(one) === 'string') {
```

Comment:
Inputs here are either an ExpressionNode, a string, or a jsnumber. If a jsnumber, we convert to an ExpressionNode just for the purpose of getting a token list. If a string, we manually generate a token. Another option might be to treat strings as if they were a var (i.e. also create an ExpressionNode to generate a token list). This would work, but is a little duplicitous since the types of strings we'd be doing that with are stuff like "= ". In any case, it seems there's opportunity to clean up this function and make things a little clearer.